### DISCUSSION TITLE
Kanban view [Apr 07]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1681] [Discord Threads]
> 🙋‍♂️ **ofir**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### ofir@discord Kanban & Calendar are work in progress, in our pipeline.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/961196703996854313)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
